powdr/guest/src/main.rs

```diff
@@ -0,0 +1,14 @@
+use lib::{Processor, ZkvmProcessor};
+
+use powdr_riscv_runtime;
+
+fn main() {
+    let input = Processor::get_guest_inputs().expect("[powdr] couldn't load inputs");
+    let output = Processor::prove(input).expect("[powdr] error during proving");
+
+    powdr_riscv_runtime::io::write(3, &output);
```

Comment:
for now powdr requires file descriptors for reads and writes, can update after it supports streams too